README.md

```diff
@@ -47,6 +47,7 @@ New tools are being added frequently, so check this page again!
 | TSX                    | [Prettier]            | [ESLint]                         |
 | TypeScript             | [Prettier]            | [ESLint]                         |
 | YAML                   | [yamlfmt]             |                                  |
+| Everything             |                       | [keep-sorted]                    |
```

Comment:
Isn't this more a formatter than a linter?

Reply:
There's no way I could see to add an Everything formatter given the current setup; since it's 1 tool to 1 language
keep-sorted has a fix and a lint mode; so it definitely works with lint as well. 🙃 
It felt like the most straightforward way to add it to the rule albeit it would be nice to add it as a formatter too.

happy to take suggestions.

Reply:
changing the order of lines can have side-effects, so it's not a formatter. As the top of the README explains, formatters should be blindly applyable without alternatives or testing.